.readthedocs.yaml

```diff
@@ -11,6 +11,11 @@ build:
     post_checkout:
       - git fetch --tags
 
+    # Replace versions in sponsor URLs
+    pre_build:
+      - python -Im pip install tox-uv
+      - python -Im tox run -e docs-sponsors
```

Comment:
Here's a few examples w/ tox/nox/scripts I got, with more involved integrations, FYI:

- https://github.com/ansible/awx/blob/587f0ec/.readthedocs.yaml
- https://github.com/ansible/docsite/blob/3ae5a22/.readthedocs.yaml
- https://github.com/ansible/jinja-docsite/blob/00290a6/.readthedocs.yaml
- https://github.com/ansible/ansible-documentation/blob/780b4d9/.readthedocs.yaml

Not something to implement in this PR, but maybe at some point. One of the things to do in CIs would be running tox twice. First, with `--notest` and then, with `--skip-pkg-install`. This makes sure that two stages are represented as separate inspectable entries in the log.